packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -44,7 +44,11 @@ export class DocumentLoad extends BasePlugin<unknown> {
    * callback to be executed when page is loaded
    */
   private _onDocumentLoaded() {
-    this._collectPerformance();
+    // Timeout is needed as load event doesn't have yet the performance metrics for loadEnd.
+    // Support for event "loadend" is very limited and cannot be used
+    window.setTimeout(() => {
+      this._collectPerformance();
+    }, 1);
```

Comment:
Does this need to be 1? Could it just be omitted instead?
```js
window.setTimeout(() => {
  this._collectPerformance();
});
```

Reply:
it can be 0 it can be omitted, in the end in most cases the browser will set it for 4ms anyway, I had previously 100 and then deleted just zeros :)